lib/api/src/js/instance.rs

```diff
@@ -112,29 +112,40 @@ impl Instance {
     /// Is expected that the function [`Instance::init_envs`] is run manually
     /// by the user in case the instance has any Wasmer imports, so the function
     /// environments are properly initiated.
-    pub fn from_module_and_instance(module: &Module, instance: WebAssembly::Instance) -> Self {
+    #[doc(hidden)]
```

Comment:
Why is this hidden? Should it be a private (non-pub) function instead? Is this actually safe to call or does the lack of env initialization make it potentially unsound?